src/components/user/appointments/UserAppointments.js

```diff
@@ -6,12 +8,24 @@ import {
   StyledSearchForm, StyledSearchHeader, StyledSearchItems, StyledSearchList,
   StyledTitle,
 } from './UserAppointmets.styled';
+import SuccessIcon from '../../../assets/notification_success.svg';
+import { getAppointments } from '../../../store/user/patientOperations';
 import Appointment from './Appointment';
-import useRequest from '../../../hooks/useActions';
+import useActions from '../../../hooks/useActions';
+import { NotifySuccess } from '../../UI/Notify';
 
 const UserAppointments = () => {
-  const { appointments } = useRequest();
-
+  const [appointments, setAppointments] = useState([]);
+  const { isAdded, isAddedChangeState } = useActions();
+  useEffect(() => {
+    getAppointments(0, 100).then((response) => setAppointments(response.data.appointments));
+  }, []);
+  if (isAdded) {
+    toast.success('Appointment successfully added', {
+      icon: () => <img src={SuccessIcon} alt="icon" />,
+    });
+    isAddedChangeState();
```

Comment:
What is this for?

Reply:
My adding appointment logic link with redux. I Have state calls 'isAdded'. When I add my appointment, I change my 'isAdded' state to true, then redirect. So, to change my state 'isAdded' to false, I use function that change it in redux. I did it in order to patient be able to add 1 more appointment and will have notification that it is added) If it not clear , I would explain it in conversation)